<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IPAddressCtrl</class>
 <widget class="QWidget" name="IPAddressCtrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="IpLineEdit" name="ipEd0"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>.</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="IpLineEdit" name="ipEd1"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>.</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="IpLineEdit" name="ipEd2"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>.</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="IpLineEdit" name="ipEd3"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IpLineEdit</class>
   <extends>QLineEdit</extends>
   <header>iplineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
